specs/eventing/README.md

```diff
@@ -0,0 +1,25 @@
+# Knative Eventing API spec
+
+This directory contains the specification of the Knative Eventing API, which is
+implemented in [`eventing.knative.dev`](/pkg/apis/eventing/v1beta1) and verified
```

Comment:
since we moved to different repo, we should probably give the full link here and also change to v1 since that's what we're targeting?
https://github.com/knative/eventing/tree/main/pkg/apis/eventing/v1

Reply:
fixed